src/main/kotlin/no/nav/familie/ba/sak/saksstatistikk/SaksstatistikkScheduler.kt

```diff
@@ -0,0 +1,54 @@
+package no.nav.familie.ba.sak.saksstatistikk
+
+import no.nav.familie.ba.sak.saksstatistikk.domene.SaksstatistikkMellomlagringRepository
+import no.nav.familie.ba.sak.saksstatistikk.domene.SaksstatistikkMellomlagringType
+import no.nav.familie.ba.sak.vedtak.producer.KafkaProducer
+import no.nav.familie.leader.LeaderClient
+import org.slf4j.LoggerFactory
+import org.springframework.scheduling.annotation.Scheduled
+import org.springframework.stereotype.Component
+import org.springframework.transaction.annotation.Transactional
+import java.time.LocalDateTime
+
+@Component
+class SaksstatistikkScheduler(
+    val saksstatistikkMellomlagringRepository: SaksstatistikkMellomlagringRepository,
+    val kafkaProducer: KafkaProducer
+) {
+
+
+    @Scheduled(fixedDelay = 60000)
+    fun sendKafkameldinger() {
+        if (LeaderClient.isLeader() == true) {
+            sendSaksstatistikk()
+        }
+    }
+
+    @Transactional
+    fun sendSaksstatistikk() {
```

Comment:
```suggestion
    fun sendStatistikk() {
```

Sender ikke denne begge typer statistikk?

Reply:
Sender bare saksstatistikk. Stønadsstatistikk er taskbasert.